plugins/woocommerce/includes/class-wc-comments.php

```diff
@@ -59,6 +59,9 @@ public static function init() {
 		// Exclude product reviews from general comments.
 		add_filter( 'comments_clauses', array( ReviewsUtil::class, 'comments_clauses_without_product_reviews' ), 10, 2 );
 
+		// Initialize ReviewsUtil hooks.
+		ReviewsUtil::init();
```

Comment:
Why did you decide to use a different approach here with a new static method?
In my opinion, it would be better to use the same standard as for the other filters. For example, the `comments_clauses` filter.
The wrapping method hides the used filter in the ReviewUtil class, but the other filters are used directly from the `WC_Comments`.